spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -82,7 +82,9 @@ object SpdxLicenseAliasMapping {
         "GPL2" to GPL_2_0_ONLY,
         "gpl3" to GPL_3_0_ONLY,
         "GPLv2" to GPL_2_0_ONLY,
+        "GPLv2+" to GPL_2_0_OR_LATER,
```

Comment:
Hmm. I'm currently wondering whether we should put these into the `deprecatedLicenses` mapping instead. But this would also need to be done for several other existing mappings then for consistency.

Reply:
GPLv2 and GPLv3 and not valid deprecated SPDX license names so using `deprecatedLicenses` is not right place.

Reply:
"GPLv2+" and "GPLv3+" are both deprecated, however `deprecatedLicenes` only contains valid deprecated ids like "GPL-2.0+" so far, so I think "GPLv2+" fits into the custom license names mapping. This could be split into separate maps for deprecated and non-deprecated ids if it's worth it, but this should happen in a separate change.

Reply:
Yes, that's the proper rationale: The point is that `deprecatedLicenses` actually does not contain "deprecated licenses" but "deprecated license (SPDX) *names*". As such we should probably rename `deprecatedLicenses` to `deprecatedLicenseNames` afterwards.

Reply:
See https://github.com/heremaps/oss-review-toolkit/pull/2100.